.github/workflows/atqa-framework-on-checks-branch.yml

```diff
@@ -0,0 +1,195 @@
+name: Run keep accounts tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  keep-accounts-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get sources with full history
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      #Set up master with tag
+
+      #      - name: Find current Git tag
+      #        run: echo ::set-output name=VALUE::$(git describe --abbrev=0 --tags `git rev-list --tags --max-count=1`)
+      #        id: currentTag
+      #
+      #      - name: Get sources for current tag
+      #        uses: actions/checkout@v3
+      #        with:
+      #          ref: '${{ steps.currentTag.outputs.VALUE }}'
```

Comment:
Is that still needed?

Reply:
yes, we replace prevTag on currentTag when this branch will come to master